speech/speech.bas

```diff
@@ -1,33 +1,63 @@
 #INCLUDE "../common/welcome.inc"
 
-Intro$="Welcome to the Colour Maximite 2"
-Prompt$="Please enter something for me to speak, or Q to exit"
-
 PhoneticOn=0
 Speed=72
 Pitch=64
 
-MODE 3,8
 CLS
 we.clear_keyboard_buffer()
 
-PRINT "Speech - Instructions"
-PRINT
-PRINT "Enter something for the computer to say, or one of these commands:"
-PRINT "Q - Quit"
-PRINT "*phonetic on|off"
-PRINT "*speed <value> - choose a speed from 1-255"
-PRINT "*pitch <value> - choose a pitch from 1-255"
-PRINT "*config - to list the current parameters"
-PRINT
-
-PLAY TTS Intro$ + " " + Prompt$
+SUB Instructions1
+  PRINT "Speech - Instructions"
+  PRINT
+  PRINT "Enter something for the computer to say, or one of these commands:"
+  PRINT "Q - Quit"
+  PRINT "*phonetic on|off"
+  PRINT "*speed <value> - choose a speed from 1-255"
+  PRINT "*pitch <value> - choose a pitch from 1-255"
+  PRINT "*config - to list the current parameters"
+  PRINT "*help - full instructions"
+  PRINT
+END SUB
+
+SUB Instructions2
+  PRINT "Commands and speech strings can use any capital or lowercase letters."
+  PRINT 
+  PRINT "In PHONETIC ON mode"
+  PRINT 
+  PRINT "Phonetic codes:"
+  PRINT "  Vowel sounds: IY IH EH AE AA AH AO OH UH UX ER AX IX"
+  PRINT "  Dipthongs:    EY AY OY AW OW UW"
+  PRINT "  Consonants:   R L W WH Y M N NX B D G J Z ZH V DH"
+  PRINT "                S SH F TH P T K CH /H
+  PRINT "  Other:        YX WX RX LX /X DX UL UM UN
+  PRINT
+  PRINT "Stresses:"
+  PRINT "1 2 3 4 5 6 7 8"
+  PRINT
+  PRINT "an example is: WEL4LKUM tuw mae4ksihmay7t"
+  PRINT "(to say Welcome to Maximite)
+  PRINT
+END SUB
+
+Instructions1
+
+Intro$="Welcome to the Colour Maximite 2.
+Prompt$="Please enter something for me to speak, or Q to exit"

```

Comment:
Should be "something for me to say", not "something for me to speak".

Also should the prompt just be something simple like "SAY >>" as the current prompt takes up half the screen?

Reply:
Changed to "Say".
I like reminding about how to quit, in case the user wasn't paying attention at the start when the instructions were shown. I've put a short input prompt on a new line, separate from the long prompt.

Reply:
In the future perhaps we can improve the UI so that the instructions regarding quit are always displayed at the top or bottom without impinging on the prompt.